client/src/app.js

```diff
@@ -327,6 +342,12 @@ export default function main({ DOM, HTTP, route, storage, scanner: scan$, search
     , !process.env.ASSET_MAP_URL ? O.empty() : O.of(
                                 { category: 'asset-map',  method: 'GET', path: process.env.ASSET_MAP_URL, bg: true })
 
+    //fetch asset List 
+    , goAssetList$.map(d => {
+      return ({ category: 'assetlist',  method: 'GET'
+          , path: `${process.env.ASSETLIST_URL}?sort_field=${d.sortField}&sort_dir=${d.sortDir}&limit=${d.limit}&start_index=${d.startIndex}` })
```

Comment:
The string arguments (`sortField` and `sortDir`) need to be URL-escaped, which can be done with `encodeURIComponent()`.

Alternatively, you can pass the query string parameters as an object, which will automaticlaly take care of escaping it, and also be more readable. Like this

```
goAssetList$.map(d => ({ category: 'assetlist',  method: 'GET', path: 'assets/registry'
                       , query: { sort_field: d.sortField, sort_dir: d.sortDir, limit: d.limit, start_index: d.startIndex } }))
```

Reply:
changes have been made